turbo.json

```diff
@@ -2,26 +2,13 @@
   "$schema": "https://turborepo.org/schema.json",
   "pipeline": {
     "build": {
-      "dependsOn": ["^build"],
       "outputs": ["dist/**"]  
     },
-    "test": {
-      "dependsOn": [],
-      "outputs": []
-    },
-    "lint": {
-      "dependsOn": ["^lint"],
-      "outputs": []
-    },
-    "typescript": {
-      "dependsOn": ["^typescript"],
-      "outputs": []
-    },
-    "extract-translations": {
-      "outputs": []
-    },
-    "dev": {
-      "cache": false
-    }
+    "coverage": {},
+    "extract-translations": {},
+    "lint": {},
+    "test": {},
+    "test:watch": {},
+    "typescript": {}
```

Comment:
Looks like I wasn't using topological dependencies in pipelines correctly. I've amended these pipelines accordingly.

Reply:
I would've thought that toplogical dependencies would be required for typescript and _maybe_ linting (since the linter might also check types)?

Reply:
Yeah, I went a bit overboard with that. I've amended things in the latest commit.